tests/integration/ceph_smoke_test.go

```diff
@@ -94,6 +94,7 @@ func (s *SmokeSuite) SetupSuite() {
 		ConnectionsCompressed:     true,
 		UseCrashPruner:            true,
 		EnableVolumeReplication:   true,
+		EnableCSINFS:              true,
```

Comment:
I think `TestNFSCSI` is a more clear statement regarding the test suite behavior than `EnableCSINFS`.

```suggestion
		TestNFSCSI:              true,
```